pylib/gyp/__init__.py

```diff
@@ -24,6 +24,17 @@
 DEBUG_VARIABLES = "variables"
 DEBUG_INCLUDES = "includes"
 
+def EscapeForCString(string):
+    if isinstance(string, str):
+        string = string.encode(encoding='utf8')
+
+    result = ''
+    for char in string:
+        if not (32 <= char < 127) or char in (ord('\\'), ord('"')):
```

Comment:
Type hints and set lookup is superfast in Python.
```suggestion
def EscapeForCString(string: bytes | str) -> str:
    if isinstance(string, str):
        string = string.encode(encoding='utf8')

    backslash_or_double_quote = {ord('\\'), ord('"')}
    result = []
    for char in string:
        if char in backslash_or_double_quote or not 32 <= char < 127:
```

Reply:
@cclauss thank you, updated! 🙂